gemrb/core/DisplayMessage.h

```diff
@@ -63,46 +57,49 @@ class GEM_EXPORT DisplayMessage
 	/** displays a string constant in the textarea */
-	void DisplayConstantString(size_t stridx, const Color &color, Scriptable *speaker = nullptr) const;
-	/** displays actor name - action : parameter */
-	void DisplayConstantStringNameString(size_t stridx, const Color &color, size_t stridx2, const Scriptable *actor) const;
+	void DisplayConstantString(size_t stridx, unsigned char color, Scriptable *speaker = nullptr) const;
 	/** displays a string constant followed by a number in the textarea */
-	void DisplayConstantStringValue(size_t stridx, const Color &color, ieDword value) const;
-	/** displays a string constant in the textarea, starting with speaker's name */
+	void DisplayConstantStringValue(size_t stridx, unsigned char color, ieDword value) const;	
+	/** displays actor name - action : parameter */
+	void DisplayConstantStringName(size_t stridx, unsigned char color, const Scriptable *speaker) const;
 	void DisplayConstantStringName(size_t stridx, const Color &color, const Scriptable *speaker) const;
-	/** displays a string constant in the textarea, starting with speaker's name, also replaces one numeric value (it is a format string) */
-	void DisplayConstantStringNameValue(size_t stridx, const Color &color, const Scriptable *speaker, int value) const;
 	/** displays a string constant in the textarea, starting with actor, and ending with target */
-	void DisplayConstantStringAction(size_t stridx, const Color &color, const Scriptable *actor, const Scriptable *target) const;
+	void DisplayConstantStringAction(size_t stridx, unsigned char color, const Scriptable *actor, const Scriptable *target) const;	
+	/** displays a string in the textarea, starting with speaker's name */
+	void DisplayStringName(ieStrRef stridx, unsigned char color, const Scriptable *speaker, STRING_FLAGS flags) const;
+	void DisplayStringName(String text, unsigned char color, const Scriptable *speaker) const;
+	void DisplayStringName(String text, const Color &color, const Scriptable *speaker) const;
+	void DisplayStringName(ieStrRef stridx, const Color &color, const Scriptable *speaker, STRING_FLAGS flags) const;
+	/** displays a string constant in the textarea, starting with speaker's name, also replaces one numeric value (it is a format string) */
+	void DisplayConstantStringNameValue(size_t stridx, unsigned char color, const Scriptable *speaker, int value) const;
+	/** displays actor name - action : parameter */
+	void DisplayConstantStringNameString(size_t stridx, unsigned char color, size_t stridx2, const Scriptable *actor) const;
 	/** displays a string in the textarea */
 	void DisplayString(const String& text) const;
+	void DisplayString(ieStrRef stridx, unsigned char color, STRING_FLAGS flags) const;
+	void DisplayString(const String& text, unsigned char color, Scriptable *target) const;
 	void DisplayString(ieStrRef stridx, const Color &color, STRING_FLAGS flags) const;
 	void DisplayString(String text, const Color &color, Scriptable *target) const;
```

Comment:
I think this one can be private now.

Reply:
I found this ones using those with git grep.

```
gemrb/core/GameScript/GSUtils.cpp:                                      displaymsg->DisplayString(sb.text);
gemrb/core/GameScript/GSUtils.cpp:              displaymsg->DisplayString(ieStrRef(parameters->int0Parameter), DMC_BG2XPGREEN, STRING_FLAGS::SOUND);
gemrb/core/GameScript/GameScript.h:     static void DisplayString(Scriptable* Sender, Action* parameters);
gemrb/core/Interface.cpp:                       if (feedback) displaymsg->DisplayString(str, DMC_WHITE, STRING_FLAGS::NONE);
gemrb/core/Map.cpp:                     displaymsg->DisplayString(RestHeader.Strref[idx], DMC_GOLD, STRING_FLAGS::SOUND);
gemrb/core/Scriptable/Actor.cpp:                                displaymsg->DisplayString(text, DMC_BG2XPGREEN, this);
gemrb/plugins/GUIScript/GUIScript.cpp:          displaymsg->DisplayString(usedItem.value, DMC_WHITE, STRING_FLAGS::SOUND);
gemrb/plugins/GUIScript/GUIScript.cpp:static PyObject* GemRB_DisplayString(PyObject * /*self*/, PyObject* args)
gemrb/plugins/GUIScript/GUIScript.cpp:          displaymsg->DisplayString(strref, ColorFromPy(pycol), STRING_FLAGS::SOUND);
gemrb/plugins/GUIScript/GUIScript.cpp:                          displaymsg->DisplayString(std::move(*msg), DMC_WHITE, nullptr);
gemrb/plugins/GUIScript/GUIScript.cpp:                          displaymsg->DisplayString(L"Error: " + *error, DMC_RED, NULL);
```

Reply:
same comment applies as above, I don't see any external users of `DisplayString(String text, const Color &color, Scriptable *target)`.

Reply:
Yes I moved 
`void DisplayString(String text, const Color &color, Scriptable *target) const;`
To be private.